README.md

```diff
@@ -159,6 +159,143 @@ python ./scripts/train_hf_ner.py ./data/annotations/train.spacy ./data/annotatio
 
 The resulting model will be saved to the `hf-ner-model/` directory.
 
+## `textcat.openai.correct`: Textcat annotation with zero- or few-shot learning
```

Comment:
Bit of a nitpick, but could we maybe add a screenshot of the user interface?

Reply:
Not a nitpick! Updated the PR to add images :)